tools/admin/wskadmin

```diff
@@ -116,9 +119,10 @@ def parseArgs():
     propmenu = subparsers.add_parser('syslog', help='work with system logs')
     subparser = propmenu.add_subparsers(title='available commands', dest='subcmd')
 
-    subcmd = subparser.add_parser('get', help='get logs for a transaction id')
-    subcmd.add_argument('tid', help='the transaction id')
-    subcmd.add_argument('component', help='component, one of [controller, invokerN] where N is invoker index')
+    subcmd = subparser.add_parser('get', help='get logs')
```

Comment:
if you do 

```
git fetch --all 
git rebase upstream/master
```
then you pick up the latest changes without making them appear as new changes in the pr.
@houshengbo wrote up a nice tutorial that might be helpful in general.